src/format/locales.rs

```diff
@@ -1,41 +1,93 @@
-use pure_rust_locales::{locale_match, Locale};
+#[cfg(feature = "unstable-locales")]
+mod localized {
+    use pure_rust_locales::{locale_match, Locale};
 
-pub(crate) const fn default_locale() -> Locale {
-    Locale::POSIX
-}
+    pub(crate) const fn default_locale() -> Locale {
+        Locale::POSIX
+    }
 
-pub(crate) const fn short_months(locale: Locale) -> &'static [&'static str] {
-    locale_match!(locale => LC_TIME::ABMON)
-}
+    pub(crate) const fn short_months(locale: Locale) -> &'static [&'static str] {
+        locale_match!(locale => LC_TIME::ABMON)
+    }
 
-pub(crate) const fn long_months(locale: Locale) -> &'static [&'static str] {
-    locale_match!(locale => LC_TIME::MON)
-}
+    pub(crate) const fn long_months(locale: Locale) -> &'static [&'static str] {
+        locale_match!(locale => LC_TIME::MON)
+    }
 
-pub(crate) const fn short_weekdays(locale: Locale) -> &'static [&'static str] {
-    locale_match!(locale => LC_TIME::ABDAY)
-}
+    pub(crate) const fn short_weekdays(locale: Locale) -> &'static [&'static str] {
+        locale_match!(locale => LC_TIME::ABDAY)
+    }
 
-pub(crate) const fn long_weekdays(locale: Locale) -> &'static [&'static str] {
-    locale_match!(locale => LC_TIME::DAY)
-}
+    pub(crate) const fn long_weekdays(locale: Locale) -> &'static [&'static str] {
+        locale_match!(locale => LC_TIME::DAY)
+    }
 
-pub(crate) const fn am_pm(locale: Locale) -> &'static [&'static str] {
-    locale_match!(locale => LC_TIME::AM_PM)
-}
+    pub(crate) const fn am_pm(locale: Locale) -> &'static [&'static str] {
+        locale_match!(locale => LC_TIME::AM_PM)
+    }
 
-pub(crate) const fn d_fmt(locale: Locale) -> &'static str {
-    locale_match!(locale => LC_TIME::D_FMT)
-}
+    pub(crate) const fn d_fmt(locale: Locale) -> &'static str {
+        locale_match!(locale => LC_TIME::D_FMT)
+    }
 
-pub(crate) const fn d_t_fmt(locale: Locale) -> &'static str {
-    locale_match!(locale => LC_TIME::D_T_FMT)
-}
+    pub(crate) const fn d_t_fmt(locale: Locale) -> &'static str {
+        locale_match!(locale => LC_TIME::D_T_FMT)
+    }
 
-pub(crate) const fn t_fmt(locale: Locale) -> &'static str {
-    locale_match!(locale => LC_TIME::T_FMT)
+    pub(crate) const fn t_fmt(locale: Locale) -> &'static str {
+        locale_match!(locale => LC_TIME::T_FMT)
+    }
+
+    pub(crate) const fn t_fmt_ampm(locale: Locale) -> &'static str {
+        locale_match!(locale => LC_TIME::T_FMT_AMPM)
+    }
 }
+#[cfg(feature = "unstable-locales")]
+pub(crate) use localized::*;
+#[cfg(feature = "unstable-locales")]
+pub use pure_rust_locales::Locale;
```

Comment:
Maybe make this `pub use` inside the module? Seems cleaner.

Reply:
The module is only `pub(crate)`, while this re-export should be `pub`.

Reply:
Have you tried it? I would expect that you can have more-public items inside a less-public module, as long as there's a more-public export.

Reply:
This was the error:
```
error[E0365]: `Locale` is only public within the crate, and cannot be re-exported outside
  --> src\format\mod.rs:61:9
   |
61 | pub use locales::Locale;
   |         ^^^^^^^^^^^^^^^ re-export of crate public `Locale`
   |
   = note: consider declaring type or module `Locale` with `pub`

error[E0365]: `Locale` is only public within the crate, and cannot be re-exported outside
   --> src\lib.rs:432:13
    |
432 |     pub use crate::Locale;
    |             ^^^^^^^^^^^^^ re-export of crate public `Locale`
    |
    = note: consider declaring type or module `Locale` with `pub`

error[E0365]: `Locale` is only public within the crate, and cannot be re-exported outside
   --> src\lib.rs:462:9
    |
462 | pub use format::Locale;
    |         ^^^^^^^^^^^^^^ re-export of crate public `Locale`
    |
    = note: consider declaring type or module `Locale` with `pub`

For more information about this error, try `rustc --explain E0365`.
error: could not compile `chrono` (lib) due to 3 previous errors
```

Reply:
Okay, fair enough!